build.gradle.kts

```
@@ -0,0 +1,37 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "3.0.4"
+    id("io.spring.dependency-management") version "1.1.0"
+    kotlin("jvm") version "1.7.22"
+    kotlin("plugin.spring") version "1.7.22"
+    kotlin("plugin.jpa") version "1.7.22"
+}
+
+group = "com.flab"
```

`flab` 까지 하는 것보단 우리 프로젝트명까지 기입하는 것이 더 좋을 것 같기는 합니다..

```suggestion
group = "com.flab.inqueue"

---

다시 생각해보니 com.flab.inqueue 로 수정하는 것이 좋을것 같습니다.